app/src/main/java/com/github/se/wanderpals/ui/screens/CreateTrip.kt

```diff
@@ -123,13 +128,23 @@ fun CreateTrip(tripViewModel: CreateTripViewModel, nav: NavigationActions) {
             horizontalAlignment = Alignment.CenterHorizontally) {
               OutlinedTextField(
                   value = name,
-                  onValueChange = { name = it },
+                  onValueChange = { newTitle ->
+                    if (newTitle.length <= MAX_TITLE_LENGTH) {
+                      name = newTitle
+                    }
```

Comment:
can create an else case for the case when newTitle.length > MAX_TITLE_LENGTH

Reply:
Here an else is not necessary since we just want to limit the maximum possible size, (i.e. if the title length is higher than 35, don't update)

Reply:
Also note that there is no need to advertise the user that he has reached the maximum length for the title. Since there is already a little text showing the title length as a fraction

Reply:
Yes, I didn't see the character count `"${name.length}/$MAX_TITLE_LENGTH"` with `MAX_TITLE_LENGTH = 35`, because it was a bit low i.e. it was much closer to the Budger field. We (one of the team members) can move it a bit higher to the Title field later :D